src/components/EventMap.js

```diff
@@ -14,6 +16,8 @@ export default function(store){
     data: {
       mapRef: null,
       eventsLayer: null,
+      initialCoordinates: [-96.9 , 37.8],
+      initialZoom: 4
```

Comment:
Oh! Ha ok cool. Yes, in chrome, I do still see the oversized markers on the up to date branch. Zoom level is slightly too zoomed in (plus the miscentering). And I see the same issues with sidebar both open and closed.

I wonder why we see different things 🤔